src/dom-to-image-more.js

```diff
@@ -249,6 +249,7 @@
         const filter = options.filter;
         if (
             node === sandbox ||
+            util.isHTMLScriptElement(node) ||
```

Comment:
We made a conscious decison to kill `SCRIPT` elements to better freeze the DOM, but we would also benefit from killing `STYLE` and `LINK`. ([The people who worship Shadow DOM](https://github.com//lit/lit-element/issues/553) like to flood the shadow element with these tags and call it "encapsulation". </scathe>)

- The `STYLE` tags duplicate existing styling data and also add `@media` queries that shouldn't be there (because they change the cloned output in bad ways when the browser window is resized). These media queries really get in the way of debugging SVG output files.
- We can also remove `LINK` elements seeing as they are broken by the cloning process.

Thoughts @IDisposable?

Reply:
Yes, we certainly ought to not clone `STYLE` tags as they pollute our emitted SVG to no effect. Never seen a `LINK` in the wild with this either, so I'm good with skipping those as well.